dev-requirements.txt

```diff
@@ -1,6 +1,6 @@
-dash_core_components==0.12.0
+dash_core_components==0.33.0
 dash_html_components==0.11.0rc5
-dash==0.18.3
+dash==0.28.0
```

Comment:
I think it would be better if it was unlocked with >=0.18.3 so we don't have to update the version and can see if an update to dash break the renderer. Same with dcc, but html need the locked rc version for py37 compat.

Reply:
Not `dash>=0.28.0` then, in that case?